node-tests/unit/engine-addon-test.js

```diff
@@ -7,7 +7,7 @@ describe('engine-addon', function() {
   describe('updateFastBootManifest', function() {
     it('adds necessary vendorFiles to the manifest when lazyLoading is enabled', function() {
 
-      /*eslint-disable*/
+      /*eslint-disable ember/avoid-leaking-state-in-ember-objects*/
```

Comment:
If you want to simply this you should be able to use `// eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects` directly above the offending `lazyLoading` line. Then you don't to re-enable eslint below.

Reply:
Yes! use `lazyLoadind: true`

Reply:
That's better here for sure 👍. I was also thinking of the one on line 34 which has `lazyLoading` disabled; can you do that without using the `enabled` property?

Reply:
`lazyLoading: true` causes the unit test to fail

Reply:
No problem @mattmarcum